tests/foreman/ui/test_contenthost.py

```diff
@@ -89,6 +90,13 @@ def setUpClass(cls):
             'activationkey-id': cls.activation_key.id,
         }, force_manifest_upload=True)
         setup_org_for_a_custom_repo({
+            'url': FAKE_0_YUM_REPO,
```

Comment:
any particular reason to have two custom repos in the module?

Reply:
`FAKE_0` repo was replaced by `FAKE_6` in https://github.com/SatelliteQE/robottelo/commit/84c0d408c18d950079bc914616d5a1b2eba88809#diff-cd3da9925eb96fb6a71086b5e30e0dd9L77
But that repo does not contain package groups needed for package group tests.
Cast @abalakh Why that repo URL was replaced and is it possible to use only one?

Reply:
Hard to say, it's been over a year. From what i remember, 0_yum_repo was not working with one of errata scenarios tested, can't provide more details - it could be issues with unexpected result e.g. errata apply wasn't applying anything (returning no applicable errata message) or issue with repo's errata itself like wrong id or smth. It's definitely possible to use only one, you'll just have to retest the module - if all the tests are passing then we are good. At that specific point 0_yum_repo was giving us some failures while 6_yum_repo was stable, that basically it.

Reply:
Ok, with zoo repo (`FAKE_0`) errata tests failed, so I guess we need both repos.